homeassistant/components/http/__init__.py

```diff
@@ -28,7 +27,7 @@
 from .const import KEY_AUTHENTICATED, KEY_HASS, KEY_HASS_USER  # noqa: F401
 from .cors import setup_cors
 from .forwarded import async_setup_forwarded
-from .request_context import setup_request_context
+from .request_context import current_request, setup_request_context
```

Comment:
Should we update uses of current request from `http.current_request` to `http.request_context.current_request`?

Reply:
I don't think so. I think that other integrations shouldn't be aware how an integration internals are structured.